operator/internal/manifests/internal/config/build.go

```diff
@@ -36,7 +36,7 @@ func Build(opts Options) ([]byte, []byte, error) {
 	w := bytes.NewBuffer(nil)
 	err := lokiConfigYAMLTmpl.Execute(w, opts)
 	if err != nil {
-		return nil, nil, kverrors.Wrap(err, "failed to create loki configuration")
+		return nil, nil, kverrors.Wrap(err, "failed to create loki configuration", err.Error())
```

Comment:
Does this make it so the error is more visible?

Reply:
Yes it does. I added this file to the PR by mistake.. but I don't mind keeping it